### DISCUSSION TITLE
Is there any way to install modelx and lifelib using pip in Ubuntu Linux?

### DISCUSSION THREAD

**Discussion by happyych** [fumitoh/modelx#124] [Q&A]
Hello, I want to use modelx in Ubuntu nowadays.
However, when I tried to install the package, the error occur with the ModulenotFound error.

When I tried other libraries like numpy or pandas, it works.

Do I have to do something before installing modelx via pip installer?

I followed the installation guide below
https://lifelib.io/quickstart/installation.html#installation

Thanks for reading my question!

**Comment by fumitoh** [comment]
modelx should be installed on Linux without problem. Can you show the error message you got?

**Reply by happyych** [reply]
![ERROR_SCREENSHOT](https://github.com/fumitoh/modelx/assets/82881835/e2ca3d0e-bf3f-4337-af23-fb4e290a5d80)


Sorry for the late reply.

Above is the code I typed for the installation.

"sudo apt  install  python3-lifelib"

and the error messege is "ModulenotFound error" in Korean.

Thanks!

**Reply by happyych** [reply]
for the additional information, I used ubuntu 22.04 verison for the test.

**Reply by fumitoh** [reply]
`apt` is for managing Linux packages, not Python packages. You should be able to use `pip` once your Python is properly installed in your Linux environment. I use [virtualenv](https://virtualenv.pypa.io/en/latest/index.html) to prepare a Python environment, and I also use [pyenv](https://github.com/pyenv/pyenv) to manage multiple Python versions installed on my Linux environment.

**Reply by happyych** [reply]
Thank you for the advice. when I tried it on virtual environment, it works!

Thanks!
